Source/WebKit/NetworkProcess/webrtc/NetworkRTCUDPSocketCocoa.mm

```diff
@@ -129,6 +129,41 @@ static dispatch_queue_t udpSocketQueue()
     m_connections->sendTo(data, address, options);
 }
 
+static int getECN(nw_content_context_t nwContext)
+{
+    // Set Default ECN value
+    rtc::EcnMarking ecn = rtc::EcnMarking::kNotEct;
+
+    nw_protocol_definition_t protocol = nw_protocol_copy_ip_definition();
+    nw_protocol_metadata_t metadata = nw_content_context_copy_protocol_metadata(nwContext, protocol);
```

Comment:
I would tend to use auto for both, and use adoptNS/adoptCF for metadata, so as to not call nw_release.
In its current form, if nw_protocol_metadata_is_ip returns false, `metadata` would leak.

Reply:
@youennf  thanks for the review. updated.